app/Nova/Team.php

```diff
@@ -61,6 +62,10 @@ public function fields(Request $request)
                 ->alwaysShow()
                 ->rules('required'),
 
+            BelongsTo::make('Project Manager', 'projectManager', User::class)
```

Comment:
Do we care about Co-PMs for this use case?

Reply:
No.